src/os/bluestore/BlueStore.cc

```diff
@@ -3760,6 +3760,11 @@ int BlueStore::_open_alloc()
   alloc = Allocator::create(cct, cct->_conf->bluestore_allocator,
                             bdev->get_size(),
                             min_alloc_size);
+  if (alloc == nullptr) {
+    lderr(cct) << "Allocator::unknown alloc type" << cct->_conf->bluestore_allocator << dendl;
```

Comment:
please add a colon and a space after  "type", and would be better to use `__func__` here. like
```c++
if (!alloc) {
  lderr(cct) << __func__ << "unknown bluestore_allocator: " << cct->_conf->bluestore_allocator << dendl;
}
```